webapp/store/views.py

```diff
@@ -155,6 +155,18 @@ def details_overview(entity_name):
     # Remove Markdown comments
     readme = re.sub("(<!--.*-->)", "", readme, flags=re.DOTALL)
 
+    show_notification = True
+
+    excluded_packages = [
+        "mattermost-charmers-mattermost",
+        "prometheus",
```

Comment:
```suggestion
        "charmcraft-prometheus",
```
my fault. I got given the wrong one....